Exercises/kokkoskernels/BlockJacobi/Begin/run_installlibs_cmake.sh

```diff
@@ -1,42 +1,40 @@
 #!/bin/bash
 
-rm -r CMakeCache.txt CMakeFiles cmake_install.cmake
+rm -r CMakeCache.txt CMakeFiles cmake_install.cmake LibInstalls
 
 KOKKOS_PATH=${HOME}/Kokkos/kokkos
 KOKKOSKERNELS_PATH=${HOME}/Kokkos/kokkos-kernels
 
+CXX_FLAG=
+KOKKOS_ARCH_FLAG=
+
 KOKKOS_DEVICES="Cuda,OpenMP"
-CXX=
-KOKKOS_ARCH=
+# Override the compiler and CPU/GPU architectures here:
+# - add value after --compiler= or --arch=
+# - uncomment the line
+#CXX_FLAG="--compiler="
+#KOKKOS_ARCH_FLAG="--arch="
 TPLS=
 OPTIONS=
 CUDA_OPTIONS=
 
-if [[ "${KOKKOS_DEVICES}" == *Cuda* ]]; then
-  CXX=${KOKKOS_PATH}/bin/nvcc_wrapper
-  KOKKOS_ARCH="BDW,Volta70"
-  CUDA_OPTIONS="enable_lambda"
-else
-  KOKKOS_ARCH="BDW"
-  CXX=g++
-fi
-
 EXERCISE_DIR=${PWD}
 mkdir -p LibInstalls
 KOKKOSKERNELS_INSTALL=${EXERCISE_DIR}/LibInstalls/kernels-install
 cd LibInstalls
 
-echo ${KOKKOSKERNELS_PATH}/cm_generate_makefile.bash --compiler=${CXX} --with-devices=${KOKKOS_DEVICES} --kokkos-path=${KOKKOS_PATH} --kokkoskernels-path=${KOKKOSKERNELS_PATH} --prefix=${KOKKOSKERNELS_INSTALL} --with-options=${OPTIONS} --with-cuda-options=${CUDA_OPTIONS} --arch=${KOKKOS_ARCH} --with-tpls=${TPLS} --kokkos-make-j=8 --disable-kokkos-tests --disable-tests --disable-examples
+CONFIG_CMD="${KOKKOSKERNELS_PATH}/cm_generate_makefile.bash $CXX_FLAG --with-devices=${KOKKOS_DEVICES} --kokkos-path=${KOKKOS_PATH} --kokkoskernels-path=${KOKKOSKERNELS_PATH} --prefix=${KOKKOSKERNELS_INSTALL} --with-options=${OPTIONS} --with-cuda-options=${CUDA_OPTIONS} $KOKKOS_ARCH_FLAG --with-tpls=${TPLS} --kokkos-make-j=8 --disable-kokkos-tests --disable-tests --disable-examples --no-default-eti"
 
-${KOKKOSKERNELS_PATH}/cm_generate_makefile.bash --compiler=${CXX} --with-devices=${KOKKOS_DEVICES} --kokkos-path=${KOKKOS_PATH} --kokkoskernels-path=${KOKKOSKERNELS_PATH} --prefix=${KOKKOSKERNELS_INSTALL} --with-options=${OPTIONS} --with-cuda-options=${CUDA_OPTIONS} --arch=${KOKKOS_ARCH} --with-tpls=${TPLS} --kokkos-make-j=8 --disable-kokkos-tests --disable-tests --disable-examples
+echo $CONFIG_CMD
+$CONFIG_CMD
 
 make install -j8
 
 cd ${EXERCISE_DIR}
 
 echo "KERNELS_INSTALL_PATH = $KOKKOSKERNELS_INSTALL"
 if [[ -d "${KOKKOSKERNELS_INSTALL}/lib64" ]]; then
```

Comment:
Thanks for all the work on this @brian-kelley !
With our CMake requirement we might be able to replace the use of `KokkosKernels_DIR` with `KokkosKernels_ROOT` and just point to the install in the run_installlibs_cmake.sh, something like replacing lines 36-40 here with
`cmake -DKokkosKernels_ROOT="${KOKKOSKERNELS_INSTALL}"` ?

I'm not requesting the change here, just noting as possible future update

Reply:
@ndellingwood Good suggestion, I just went through and changedthat